### PR TITLE
Add issue templates

### LINKED ISSUE #428: Provide playground link in readme and new issues
Apparently TypeScript’s playground allows importing from npm:

Screen Shot

URL: https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBDAnmApnA3gUQB4GMVgwC+cAZlBCHAORKoC0pKAzjNQNwCwAUD3WgDEIEOAF4MPAJABXAHbBc0WQC44rKMFkBzLt0lQAhpoBGEAO6rTEADYoDs3UV19kg4QHVgMABYRpMACUjWVMzMTgcfEIAHiEIABoaQxNzagA+XQB6TNE0jDkFJVV1TS0iIA

I think this URL should be included in the readme and other places.

## Upvote & Fund

- We're using [Polar.sh](https://polar.sh/sindresorhus) so you can upvote and help fund this issue.
- The funding will be given to active contributors.
- Thank you in advance for helping prioritize & fund our backlog.

<a href="https://polar.sh/sindresorhus/type-fest/issues/428">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/sindresorhus/type-fest/issues/428/pledge.svg?darkmode=1">
  Fund with Polar
</picture>
</a>

### PR DESCRIPTION
**Pull request by fregante**
I labelled all the unlabelled issues just now. These template would make it easier to triage new issues

- will close #428

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/2 bug report.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+name: 🐛 Report bug
+about: __________________
+labels: 'bug'
+---
```
Comment (fregante) [inline .github/ISSUE_TEMPLATE/2 bug report.md:5]: I should probably turn this into an "issue form" and ***require** a repro URL as specified on:

https://github.com/sindresorhus/type-fest/issues/428

File: .github/ISSUE_TEMPLATE/1 new type.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+name: 💡 Suggest new type
+about: __________________
+title: 'Proposal: '
+labels: 'type addition'
```
Comment (fregante) [inline .github/ISSUE_TEMPLATE/1 new type.md:5]: This should also use a "issue form" to ask to search existing requests and types first.

File: .github/ISSUE_TEMPLATE/1 new type.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+name: 💡 Suggest new type
+about: __________________
```
[the remark this answers is not in the document]
Comment (fregante) [reply .github/ISSUE_TEMPLATE/1 new type.md:3]: See the issue selector on Refined GitHub 

GitHub requires a second line to further describe the type of issue, but 3/4 words are already enough. If there are too many words on screen people won't read them anyway. 

Compare: https://github.com/eslint/eslint/issues/new/choose
To: https://github.com/refined-github/refined-github/issues/new/choose

[the remark this answers is not in the document]
Comment (Emiyaaaaa) [reply .github/ISSUE_TEMPLATE/1 new type.md:3]: Is there a way to set this to null or ""?

[the remark this answers is not in the document]
Comment (fregante) [reply .github/ISSUE_TEMPLATE/1 new type.md:3]: No: https://github.com/fregante/sandbox/blob/master/.github/ISSUE_TEMPLATE/a.md

[the remark this answers is not in the document]
Comment (fregante) [reply .github/ISSUE_TEMPLATE/1 new type.md:3]: Best I could do is moving the emojis there, but it doesn't look as good


<table>
<tr>
	<th>Before
	<th>After
<tr>
	<td>Screenshot 3
	<td>Screenshot 4
</table>

[the remark this answers is not in the document]
Comment (fregante) [reply .github/ISSUE_TEMPLATE/1 new type.md:3]: It looks like one of these characters worked 🙂 

```html
&#8203;&#8204;&#8205;&#65279;&#8288;&#8287;
```

[the remark this answers is not in the document]
Comment (Emiyaaaaa) [reply .github/ISSUE_TEMPLATE/1 new type.md:3]: this is look better

**Review by voxpelli — APPROVED** [review]
Seems like a good level for issue forms 👍

And normal issues are still available, right? So that eg we can create issues for stuff like this still :)

File: .github/ISSUE_TEMPLATE/1 new type.yml
Comment (fregante) [inline .github/ISSUE_TEMPLATE/1 new type.yml:1]: Screenshot

File: .github/ISSUE_TEMPLATE/3 enhancement.md
Comment (fregante) [inline .github/ISSUE_TEMPLATE/3 enhancement.md:1]: Screenshot

File: .github/ISSUE_TEMPLATE/2 bug report.yml
Comment (fregante) [inline .github/ISSUE_TEMPLATE/2 bug report.yml:1]: Screenshot

**Review by fregante — COMMENTED** [review]
Ready

**Review by fregante — COMMENTED** [review]
Ready

**Comment by fregante** [comment]
Tested in other repo, merging. They can be tweaked as needed in a future PR.

**Comment by fregante** [comment]
The new picker: https://github.com/sindresorhus/type-fest/issues/new/choose

## Screenshot

Screenshot 2

**Comment by sindresorhus** [comment]
Looks good. Thanks for adding it.

**Comment by voxpelli** [comment]
Agree, looks good 🎉

**Pull request merged**